test/Core/PublishesAsynchronousMessagesTest.php

```diff
@@ -14,7 +14,7 @@ class PublishesAsynchronousMessagesTest extends TestCase
 {
 
     /**
-     * @var Publisher|PHPUnit_Framework_MockObject_MockObject
+     * @var Publisher|MockObject
```

Comment:
I recently learned that type correct type for these is actually `Publisher & MockObject`. 
It's not either of them, it's both.

Reply:
Thanks for pointing that out, will use the `&` operator from now on for mocked objects.